packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2035,14 +2036,14 @@ void main() {
     await tester.pumpAndSettle();
 
     // Check text location in text field.
-    expect(tester.getTopLeft(find.text('Hint')).dx, 48.0);
+    expect(tester.getTopLeft(find.text('Hint')).dx, 52.0);
 
     // By default, the text of item 0 should be aligned with the text of the text field.
     expect(tester.getTopLeft(find.text('Item 0').last).dx, 48.0);
```

Comment:
Do you know why this line is not impacted?

Reply:
Oh, weird. I don't know. But let me check.

Reply:
It was happening because I forgot to add padding when leading icon is not there. I have fixed and pushed the change.

Reply:
Great 👍